.github/workflows/package.yml

```
@@ -70,3 +70,29 @@ jobs:
         with:
           name: packages
           path: "/home/builder/groonga/*.pkg.tar.*"
+  release:
+    name: Release
+    needs: packages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v4
+      - name: Prepare release note
+        run: |
+          echo "Groonga ${GITHUB_REF_NAME}" | tee release-title.txt
+          commit_id=$(git ls-remote  https://aur.archlinux.org/groonga.git HEAD | head -n 1 | awk '{print $1}')
+          cat <<RELEASE_NOTE | tee release-note.md
+          Groonga: https://github.com/groonga/groonga/releases/tag/v${GITHUB_REF_NAME}
+          AUR: https://aur.archlinux.org/packages/groonga
+          PKGBUILD: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=groonga&id=${commit_id}
+          RELEASE_NOTE
+      - name: Create GitHub Release
+        if: github.ref_type == 'tag'
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh release create "${GITHUB_REF_NAME}" \
+            --discussion-category Announcements \
+            --notes-file release-note.md \
+            --repo "${GITHUB_REPOSITORY}" \
+            --title "$(cat release-title.txt)" \
+            packages/*
```

How about writing environments after run command like the following.
ref: https://github.com/pgroonga/pgroonga/blob/main/.github/workflows/linux.yml
```suggestion
        run: |
          gh release create "${GITHUB_REF_NAME}" \
            --discussion-category Announcements \
            --notes-file release-note.md \
            --repo "${GITHUB_REPOSITORY}" \
            --title "$(cat release-title.txt)" \
            packages/*
        env:
          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

---

Even in GitHub Actions example, `env` is often written after `run`, but I prefer to have it before for readability.

I agree with this PR:
https://github.com/groonga/groonga/pull/2313

---

Ahhh I understood. I did look for this PR and couldn’t find that.